src/components/Footer.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import { Filter } from './Filter';
+import { TodoStatus } from '../types/TodoStatus';
+
+type Props = {
+  todos: Todo[],
+  onChangeSelect: (event: TodoStatus) => void,
+  selectedOption: string,
+  onHandleDeleteAll: () => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  onChangeSelect = () => {},
+  selectedOption,
+  onHandleDeleteAll,
+}) => {
+  const counterNotCompletedTodos = todos
+    .filter((todo) => !todo.completed).length;
+
+  const isCompleted = !!todos.some((todo) => todo.completed);
```

Comment:
https://github.com/mate-academy/react_todo-app-with-api/pull/804#discussion_r1341199726 - if mentors write about some mistakes, see if you are making them somewhere else.